components/database/postgresql-12/postgresql12-contrib.p5m

```diff
@@ -0,0 +1,384 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Alexander Pyhalov
+# Copyright 2018 Michal Nowak
+#
+
+set name=pkg.fmri value=pkg:/database/postgres-12/contrib@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="PostgreSQL community contributed tools not part of core product"
+set name=info.classification value=org.opensolaris.category.2008:System/Databases
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license COPYRIGHT license="PostgreSQL"
+
+file path=usr/postgres/12/bin/oid2name
+file path=usr/postgres/12/bin/pg_archivecleanup
+file path=usr/postgres/12/bin/pg_standby
+file path=usr/postgres/12/bin/pg_test_fsync
+file path=usr/postgres/12/bin/pg_test_timing
+file path=usr/postgres/12/bin/pg_waldump
+file path=usr/postgres/12/bin/pgbench
+file path=usr/postgres/12/bin/vacuumlo
+file path=usr/postgres/12/lib/$(MACH64)/_int.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/adminpack.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/amcheck.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/auth_delay.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/auto_explain.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/autoinc.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/bloom.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/btree_gin.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/btree_gist.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/citext.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/cube.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/dblink.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/dict_int.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/dict_xsyn.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/earthdistance.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/file_fdw.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/fuzzystrmatch.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/hstore.so mode=0555
+file path=usr/postgres/12/lib/$(MACH64)/hstore_plperl.so
+file path=usr/postgres/12/lib/$(MACH64)/hstore_plpython2.so
```

Comment:
Some don't need `mode=0555`?

Reply:
None of them. They'll get mode=0555 via default transform.